### PR TITLE
add examples + refactor jss stringify from css2jss into cli

### PR DESCRIPTION
**Pull request by serapath**
I don't need the jss to be stringified when i use it inside my app. I actually need the jss, so i hope this kind of refactoring is cool with you :-)

### CODE REVIEW

**Comment by kof** [comment]
I like the changes, just a view small things and we can merge it.

File: bin/cli.js [hunk 1]
```diff
@@ -30,5 +30,14 @@ var converterExtMap = {
...
     var code = fs.readFileSync(program.sourcePath, 'utf-8')
     var ext = path.extname(program.sourcePath)
-    console.log(converterExtMap[ext](code, program))
+    var jss = converterExtMap[ext](code)
+    /*
+    * program:
+    *   - `pretty` pretty formatting
+    *   - `export` namespace string to add before output
+    */
+    var spaces    = program.pretty ? '  ' : ''
```
Comment (kof) [inline bin/cli.js]: I use normal indentation, always the same amount of spaces. Will be easier to check once we have eslint in this repository too.

File: bin/cli.js [hunk 2]
```diff
@@ -30,5 +30,14 @@ var converterExtMap = {
 ;(function convert() {
     var code = fs.readFileSync(program.sourcePath, 'utf-8')
     var ext = path.extname(program.sourcePath)
-    console.log(converterExtMap[ext](code, program))
+    var jss = converterExtMap[ext](code)
+    /*
```
Comment (kof) [inline bin/cli.js]: no need for this comment

File: example/test.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+var css = [
+  '.ninja {',
+  '  visibility: hidden;',
+  '  color: black;',
+  '}'
+].join('\n')
+
+var css2jss = require('../lib/cssToJss.js')
```
Comment (kof) [inline example/test.js]: // Would prefer the public way of using it for the purpose of education.
var cssToJss = require('..').cssToJss

File: package.json [hunk 4]
```diff
@@ -7,7 +7,8 @@
     "jss": "bin/cli.js"
   },
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "css2jss": "node bin/cli example/style.css",
+    "test": "node example/test"
```
Comment (kof) [inline package.json]: Well this is not exactly test, having tests is nice, but this is just a running example. I would suggest to either add real tests or just to remove this line.

File: example/test.js (hunk 3 above)
Comment (serapath) [inline example/test.js]: ok, i can update it, just i think, especially when using browserify, requiring just the `cssToJss` saves when it comes to file size, but it's probably not a lot anyway.

File: readme.md [hunk 5]
```diff
@@ -25,8 +25,16 @@ jss source.css -p > source.jss
...
+var css = [
+  '.ninja {',
+  '  visibility: hidden;',
+  '  color: black;',
+  '}'
+].join('\n')
+
+var css2jss = require('../lib/cssToJss.js')
+var jss = css2jss(css)
+console.log(JSON.stringify(jss, null, 2))
```
Comment (kof) [inline readme.md:38]: I think because the whole js community is moving towards es6, we should write everything in es6. I did rewrite of jss in es6 already. Examples are easy to write, no refactoring is needed.

File: readme.md [hunk 6]
```diff
@@ -25,8 +25,16 @@ jss source.css -p > source.jss
...
-import {cssToJss} from 'jss-cli'
-cssToJss(css, options)
+var css = [
+  '.ninja {',
+  '  visibility: hidden;',
+  '  color: black;',
+  '}'
+].join('\n')
+
+var css2jss = require('../lib/cssToJss.js')
```
Comment (kof) [inline readme.md]: I would keep it consistent either cssToJss or css2jss everywhere

File: package.json (hunk 4 above)
Comment (serapath) [inline package.json]: ok removed it

File: lib/cssToJss.js [hunk 7]
```diff
@@ -5,23 +5,16 @@ var css = require('css')
...
- * @param {Object} options
  * @return {String}
  * @api public
  */
-module.exports = function (code, options) {
+module.exports = function (code) {
     var ast = css.parse(code)
 
     if (ast.stylesheet && ast.stylesheet.rules) {
         var jss = toJssRules(ast.stylesheet.rules)
```
Comment (kof) [inline lib/cssToJss.js]: no need for the variable any more, you can just return it

**Comment by kof** [comment]
https://github.com/jsstyles/jss-cli/pull/3#discussion_r39318869

https://github.com/jsstyles/jss-cli/pull/3#discussion_r39318993

File: readme.md (hunk 6 above)
Comment (serapath) [inline readme.md]: updated it

File: readme.md (hunk 5 above)
Comment (serapath) [inline readme.md:38]: just that i need to add the babelify transform then - because es6 is not well supported yet and i'd like to use it in the browser. I can "es6ify" it again, sure that that is a good thing?

I always think - people who know es6, know how to rewrite, but those who don't need to learn es6 + jss, which is "one more thing".

Comment (kof) [inline readme.md:38]: I think we need to make people learn es6, its not that hard.

Comment (kof) [inline readme.md:38]: especially as es6 is javascript and lots of its features are already in spec and even in v8.

Comment (serapath) [inline readme.md:38]: yes, i just figured out, that es6 template strings are already in chrome.
ok, so i'll change the example, but in order to run anywhere, i would then also include `babelify`.

Comment (kof) [inline readme.md:38]: well lets make it the next step if you want, but then same way as jss repo.

Comment (kof) [inline readme.md:38]: for now its ok at least in documentation to use es6

Comment (kof) [inline readme.md:38]: Lets make it a separate PR with complete conversion to es6
